src/main/resources/messages.properties

```diff
@@ -268,8 +270,6 @@ timeout-notice.subtext=<p>To protect your information, your session will timeout
 language-preference.title=Language preference
 language-preference.language-to-read-label=What language do you prefer to read or write?
 language-preference.language-to-read-help=DCFS will do their best to provide documents in your preferred language.
-language-preference.language-to-speak-label=What language do you prefer to speak?
-language-preference.language-to-speak-help=DCFS will do their best to connect you with someone who speaks your language.
```

Comment:
Wasnt this already removed?

Reply:
the questions were removed but not the text strings